runtime/3rdparty/ggml/src/CMakeLists.txt

```diff
@@ -1098,7 +1098,7 @@ if (CMAKE_OSX_ARCHITECTURES      STREQUAL "arm64" OR
                 list(APPEND ARCH_FLAGS -mfpu=neon-vfpv4 -mno-unaligned-access -funsafe-math-optimizations)
             else()
                 # Raspberry Pi 2
```

Comment:
Our target is not Raspberry Pi 2.

Reply:
Please add comment about custom fix.

Ex)
https://github.com/Samsung/ONE/blob/ab777f05be9660026356cdb22519de9839409afd/runtime/3rdparty/ggml/CMakeLists.txt#L232